### PR TITLE
README badges

### LINKED ISSUE #128: Add howfairis badge and Action
Following #52, we should add a green, 5-star howfairis badge. #113 includes a CII badge, which was the only missing criterion left to get the howfairis badge from 4 to 5 stars. After #113 is merged, we can rerun howfairis to get the badge.

To make sure we stay FAIR, we can add the howfairis GitHub Action: https://github.com/fair-software/howfairis-github-action

### PR DESCRIPTION
**Pull request by egpbos**
Fixes #128, i.e. it adds a green howfairis badge now that #113 has been merged, and adds a GH Action to check this from now on.

This PR also reorders badges a in three ways:
1. Uninteresting badges are removed
2. Some top README badges are moved down to relevant sections, so that the ones at the top stand out more.
3. `badges.md` is removed, further cleaning up the repo (again following #120).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,15 +28,11 @@ affiliations:
...
 [![build](https://github.com/dianna-ai/dianna/actions/workflows/build.yml/badge.svg)](https://github.com/dianna-ai/dianna/actions/workflows/build.yml)
-[![workflow scc badge](https://sonarcloud.io/api/project_badges/measure?project=dianna-ai_dianna&metric=coverage)](https://sonarcloud.io/dashboard?id=dianna-ai_dianna)
-[![workflow pypi badge](https://img.shields.io/pypi/v/dianna.svg?colorB=blue)](https://pypi.python.org/project/dianna/)
-[![supported python versions](https://img.shields.io/pypi/pyversions/dianna)](https://pypi.python.org/project/dianna/)
-
-
 [![Documentation Status](https://readthedocs.org/projects/dianna/badge/?version=latest)](https://dianna.readthedocs.io/en/latest/?badge=latest)
+
+[![workflow scc badge](https://sonarcloud.io/api/project_badges/measure?project=dianna-ai_dianna&metric=coverage)](https://sonarcloud.io/dashboard?id=dianna-ai_dianna)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/5542/badge)](https://bestpractices.coreinfrastructure.org/projects/5542)
```
Comment (cwmeijer) [inline README.md:34]: Why not all 5 badges on one line? I'd prefer that, but I'm ok with dividing them over 2 lines although I don't really see the point of it and it takes up more space.

File: .github/workflows/fair-software.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  verify:
+    name: "fair-software"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: fair-software/howfairis-github-action@0.2.0
+        name: Measure compliance with fair-software.eu recommendations
+        env:
+          PYCHARM_HOSTED: "Trick colorama into displaying colored output" 
+        with:
+          MY_REPO_URL: "https://github.com/${{ github.repository }}"
```
Comment (cwmeijer) [inline .github/workflows/fair-software.yml:15]: Doesn't seem super easy to make it work with our current branch instead of main. bummer.

**Review by cwmeijer — APPROVED** [review]
The branch the how-fair-is action is checking (main) can certainly lead to confusing situations later on. It would be worth our time to see if we can fix this to use the current branch instead. It is already complicated enough to understand what the actions are actually checking as it is.

Great that you were able to get the 5th check for how-fair-is! Great cleanup of repo and readme!

File: README.md (hunk 1 above)
Comment (egpbos) [reply README.md:33]: Done.

**Pull request merged**